index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="styles.css"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Selcuk Guler's Personal Portfolio website</title>
+  </head>
+  <body>
+    <main>
+      <div id="topbar">
+        <div id="appbar">
+          <div id="timer-cont">
+            <div id="timer-text">9:41</div>
+          </div>
+          <div id="statusbar">
+          </div>
+        </div>
+        <div class="container" id="toolbar">
+          <div id="logo-placeholder"></div>
+          <div id="hamburger-cont">
+            <div id="hamburger-icon"></div>
+          </div>
+        </div>
+      </div>
```

Comment:
Please implement these changes for the header section of your page 👍 

- [ ] Kindly use a `header` tag for the header of your website which depicts introductory content majorly for good HTML semantics and a clean code 😄 , and usually the `header` tag comes before the `main` tag. You can check this [article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more

- [ ] Please use a `nav` tag for your hamburger menu ie, line `22 - 24` should be inside a `nav` tag 😄